styles.css

```diff
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+.main {
+  background: url("images/IMG-20220819-WA0005.jpg");
+  background-repeat: no-repeat;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 100%;
+  font-family: Poppins, Arial, Verdana, sans-serif;
+}
+
+.top {
+  display: flex;
+  height: auto;
+  flex-direction: column;
+  margin-top: -15px;
+  background-size: auto;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.header {
+  margin: auto;
+  position: center;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  font-family: Poppins, Arial, Verdana, sans-serif;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-grow: 0;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+
+.logo {
+  font-family: Poppins, Arial, Verdana, sans-serif;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  line-height: 74px;
+  text-align: center;
+  vertical-align: center;
+  color: #6070ff;
+}
+
+a {
+  color: #6070ff;
+}
+
+.burger {
+  color: #6070ff;
+}
+
+.intro {
+  font-weight: bold;
+  margin-left: 24px;
+  margin-right: 24px;
+  width: 327px;
+  height: 150px;
+  top: 114px;
+  left: 24px;
+  font-size: 30px;
+  line-height: 52px;
+  margin-top: 100px;
+  align-items: left;
+  vertical-align: center;
+  color: #172b4d;
+}
+
+.about {
+  font-size: 16px;
+  margin-left: 24px;
+  margin-right: 24px;
+  width: 327px;
+  height: 146px;
+  top: 276px;
+  left: 24px;
+  line-height: 24px;
+  align-items: left;
+  vertical-align: top;
+  color: #344563;
+  margin-top: 40px;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.description {
+  font-style: medium;
+  font-size: 10px;
+  line-height: 24px;
+  margin-left: 24px;
+  margin-right: 225px;
+  align-items: left;
+  vertical-align: top;
+  color: #7f8cff;
+  margin-top: 16px;
+}
+
+.sidebar-toggle {
+  position: relative;
+  font-size: 1rem;
+  background: transparent;
+  border-color: transparent;
+  color: #6070ff;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.icons {
+  position: relative;
+  top: 0;
+  left: 0;
+  right: 24px;
+  z-index: 1;
+  display: flex;
```

Comment:
- [ ] Please create semantic classes for your purposes when you are going to use one property over the many classes. For example, it's common to create `d-flex` class for all your flex containers. Then, if an element needs to be flex just add this class to it in HTML. An element can have multiple class names. This will make your code DRY.